.gitignore

```diff
@@ -7,6 +7,7 @@
 *.dll
 *.so
 *.dylib
+bin/
```

Comment:
nit: It will be nice adding a comment saying this directory related to the project tooling binaries, not a must.

Reply:
done.